test/e2e/schedulingbase/volume_binding.go

```diff
@@ -124,6 +126,7 @@ var _ = ginkgo.Describe("Volume Binding Test", func() {
 	)
 
 	ginkgo.BeforeEach(func(ctx context.Context) {
+		f.ClientSet = e2eutil.NewSchedulerInjectingClientset(f.ClientSet, e2eutil.SchedulerName)
```

Comment:
How about set a parameter `--scheduler-name="volcano,defaule-scheduler"` to let volcano schedule all pods directly?

Reply:
I think it may cause kube-scheduler and volcano to compete for scheduling pods and cause strange behavior, set the schedulerName directly will make sure everything is correct and scheduled by volcano